app/assets/javascripts/helpers/strings.jsx

```diff
@@ -0,0 +1,7 @@
+export function numberWithCommas(x) {
```

Comment:
Remove export: The helper function would get imported automatically, so no need for the `export` keyword